plugin/core/views.py

```diff
@@ -33,6 +33,8 @@
 import sublime_plugin
 import tempfile
 
+MarkdownLangMap = Optional[Dict[str, Tuple[Sequence[str], Sequence[str]]]]
```

Comment:
`Optional` should IMO not be a part of this type itself. It can be confusing to see a type name indicating that it's a map without being clear that it can also be `None`.